cmd/kubeadm/app/phases/bootstraptoken/node/tlsbootstrap.go

```diff
@@ -28,20 +28,11 @@ import (
 )
 
 const (
-	// NodeBootstrapperClusterRoleName defines the name of the auto-bootstrapped ClusterRole for letting someone post a CSR
-	// TODO: This value should be defined in an other, generic authz package instead of here
-	NodeBootstrapperClusterRoleName = "system:node-bootstrapper"
```

Comment:
same for this file. should all constants be moved?

Reply:
Thanks for review.  I just put together some of the built-in constants of the system. if these two constants don't fit here, I'm going to put the them back.

Reply:
I think we should probably move all these remaining constants to constants.go. Same for the proxy file.

Reply:
ok, I'm going to sort out how many constants we need to move.

Reply:
There are constants of role and rolebinding created by kubeadm, e.g: `kubeadm:kubelet-bootstrap`, `kubeadm:get-nodes`, `kubeadm:node-autoapprove-bootstrap` in following files:

[cmd/kubeadm/app/phases/bootstraptoken/node/tlsbootstrap.go
](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/bootstraptoken/node/tlsbootstrap.go)[cmd/kubeadm/app/phases/addons/proxy/proxy.go
](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/addons/proxy/proxy.go)[cmd/kubeadm/app/phases/bootstraptoken/clusterinfo/clusterinfo.go
](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/bootstraptoken/clusterinfo/clusterinfo.go)[cmd/kubeadm/app/phases/uploadconfig/uploadconfig.go
](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/uploadconfig/uploadconfig.go)

Should we move these constants to constants.go?

Reply:
makes sense to me, because constants.go already has "kubeadm:" and "system:" constants.